### PR TITLE
xeyes: update to 1.1.2

### PR DESCRIPTION
**Pull request by grimreaper**

### CODE REVIEW

File: x11/xeyes/Portfile [hunk 1]
```diff
@@ -12,9 +12,10 @@ long_description    xeyes defies description.
 homepage            https://www.x.org
 master_sites        xorg:individual/app/
 
-checksums           md5     a3035dcecdbdb89e864177c080924981 \
-                    sha1    efe6116d31a7f69e4fb6038613e52b0960b9b61c \
-                    rmd160  cc232409952e59e6d8e634c2728edde88bc4ed62
+checksums           size    133160 \
+                    sha1    458987f8accdbd8abf993ee51030391cf584d2e8 \
```
Comment (pmetzger) [inline x11/xeyes/Portfile]: I don't think sha1 is necessary at this point. Also, we traditionally put "size" last.

**Comment by pmetzger** [comment]
Merged. Thank you, @grimreaper!

**Comment by grimreaper** [comment]
sorry for not addressing your comments first. thanks!

**Comment by pmetzger** [comment]
@grimreaper If you could submit another PR to fix that, or do it directly, I would be happier. 😀

File: x11/xeyes/Portfile (hunk 1 above)
Comment (grimreaper) [reply x11/xeyes/Portfile]: dc5961f836cc268f288938ff2f32c825827613e0 done

**Pull request merged**